server/src/core/server/models/comment/comment.ts

```diff
@@ -1701,3 +1701,30 @@ export async function retrieveFeaturedComments(
 
   return results;
 }
+
+export async function retrieveLatestFeaturedCommentForAuthor(
+  mongo: MongoContext,
+  tenantID: string,
+  userID: string,
+  excludedCommentID: string
+) {
+  const $match: FilterQuery<Comment> = {
+    tenantID,
+    authorID: userID,
+    commentID: { $ne: excludedCommentID },
```

Comment:
What is the purpose of an `excludedCommentID`?

Reply:
I was thinking that way it couldn't ever grab the newly unfeatured comment here...but looking at this again, this shouldn't happen, so probably best to just simplify and remove.